src/autoscaler/routes/routes.go

```diff
@@ -28,12 +28,16 @@ const (
 
 	SyncActiveSchedulesPath      = "/v1/syncSchedules"
 	SyncActiveSchedulesRouteName = "SyncActiveSchedules"
+
+	CustomMetricsPath          = "/v1/{appid}/metrics"
```

Comment:
how about using /v1/apps/{appId}/metrics to align with the prefix of other APIs .

Reply:
Done